pydatastructs/miscellaneous_data_structures/disjoint_set.py

```diff
@@ -74,3 +77,18 @@ def union(self, key1, key2):
 
             y_root.parent = x_root
             x_root.size += y_root.size
+
+    def make_root(self, key):
```

Comment:
The feature looks good. Could you share an example (book/lecture notes examples will work) where this is used/required?

Reply:
Presently, I don't remember the exact the example but, we can consider this example https://cp-algorithms.com/data_structures/disjoint_set_union.html#toc-tgt-10 wherein we generally make root which Is largest element in set

Reply:
I see. That's great. Could you solve one such problem using DSU API in `pydatastructs` and show the code here?

Reply:
I will trying finding the exact problem I solved using this method and show that

Reply:
Well, this is the problem where I used the above method: https://codeforces.com/contest/1133/problem/F2, and the submission of mine can be found at: https://codeforces.com/contest/1133/submission/109674863